modules/core/misc/python/pyopencv_core.hpp

```diff
@@ -0,0 +1,53 @@
+#ifdef HAVE_OPENCV_CORE
+
+#include "opencv2/core/cuda.hpp"
+
+typedef std::vector<cuda::GpuMat> vector_GpuMat;
+typedef cuda::GpuMat::Allocator GpuMat_Allocator;
+
+template<> bool pyopencv_to(PyObject* o, Ptr<cuda::GpuMat>& m, const char* name);
+template<> PyObject* pyopencv_from(const Ptr<cuda::GpuMat>& m);
+
+template<>
+bool pyopencv_to(PyObject* o, cuda::GpuMat& m, const char* name)
+{
+    if (!o || o == Py_None)
+        return true;
+    Ptr<cuda::GpuMat> mPtr(new cuda::GpuMat());
+
+    if (!pyopencv_to(o, mPtr, name)) return false;
+    m = *mPtr;
+    return true;
+}
+
+template<>
+PyObject* pyopencv_from(const cuda::GpuMat& m)
```

Comment:
@alalek Can you please advice what does [this error](http://pullrequest.opencv.org/buildbot/builders/precommit_linux64_no_opt/builds/16078/steps/test_python2/logs/stdio) mean? Does that mean the file itself was not loaded?

Reply:
By the way, the unit tests I provided were skipped in all platforms, and as you know, were not tested in the custom CUDA one.